packages/suite/src/components/suite/SwitchDevice/components/DeviceItem/index.tsx

```diff
@@ -129,6 +130,8 @@ const DeviceItem = ({ device, instances, closeModalApp, backgroundRoute }: Props
     const [isExpanded, setIsExpanded] = useState(true);
     const [animateArrow, setAnimateArrow] = useState(false);
 
+    const { openNodeById } = useGuide();
```

Comment:
what about to move the hook inside OpenGuideFromTooltip component, so it has only `id` as prop and this hook  is used on one place only?

Reply:
67dcf05